src/test/kotlin/graphics/scenery/tests/examples/volumes/VolumeClient.kt

```diff
@@ -0,0 +1,109 @@
+package graphics.scenery.tests.examples.volumes
+
+import graphics.scenery.*
+import graphics.scenery.backends.Renderer
+import graphics.scenery.textures.Texture
+import graphics.scenery.ui.SwingBridgeFrame
+import graphics.scenery.utils.VideoDecoder
+import graphics.scenery.volumes.*
+import org.joml.Vector3f
+import org.joml.Vector3i
+import java.nio.ByteBuffer
+import kotlin.concurrent.thread
+
+
+class VolumeClient : SceneryBase("Volume Client", 512 , 512) {
```

Comment:
Should be renamed to something like `SimpleVolumeClient` to distinguish it from the `ClientApplication`. Also a description of what this application does should be added to the top of the class, here and in other example classes